_pages/en_US/troubleshooting.txt

```diff
@@ -420,6 +421,32 @@ If this is unsuccessful, try using another SD card.
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+## Homebrew Launcher (super-skaterhax)
+
+{% capture compat %}
+<summary><u>"An error has occured. Please save your data in any software currently in use, then restart the system."</u></summary>
+
+The date is set incorrectly. \
+To set it correctly, follow these steps: 
+1. Select the System Settings icon on the HOME Menu, and tap Open.
+1. Tap Other Settings.
+1. Tap Date & Time.
+1. Tap Today's date.
+1. Tap Up/Down Arrows to set the correct Day, Month and Year.
+1. Select OK to confirm.
+When done, go back to [Section III of Homebrew Launcher (super-skaterhax)](homebrew-launcher-(super-skaterhax)#section-iii---super-skaterhax).
```

Comment:
we don't need this "go back to previous page" thing imo. It'll make both translations awkward and the whole point of the troubleshooting page is that you will do troubleshooting and just go back to where you were